lib/p2p/seeds/index.ts

```diff
@@ -1,15 +1,41 @@
-import { default as mainnet } from './mainnet';
-import { default as testnet } from './testnet';
-import { default as simnet } from './simnet';
+import {
+  nodes as mainnetNodes,
+  currencies as mainnetCurrencies,
+  pairs as mainnetPairs,
+} from './mainnet';
+import {
+  nodes as testnetNodes,
+  currencies as testnetCurrencies,
+  pairs as testnetPairs,
+} from './testnet';
+import {
+  nodes as simnetNodes,
+  currencies as simnetCurrencies,
+  pairs as simnetPairs,
+} from './simnet';
 import * as db from '../../db/types';
 import { XuNetwork } from '../../constants/enums';
 
-const seeds: { [network: string]: db.NodeAttributes[] | undefined } = {
-  [XuNetwork.MainNet]: mainnet,
-  [XuNetwork.TestNet]: testnet,
-  [XuNetwork.SimNet]: simnet,
+const nodeSeeds: { [network: string]: db.NodeAttributes[] | undefined } = {
+  [XuNetwork.MainNet]: mainnetNodes,
+  [XuNetwork.TestNet]: testnetNodes,
+  [XuNetwork.SimNet]: simnetNodes,
+};
+
+const currencySeeds: { [network: string]: db.CurrencyAttributes[] | undefined } = {
```

Comment:
I am not sure whether `seeds` is the correct terminology for this. I always associate `seed` with a node

Reply:
Interesting - I associate it with initial seed data for the DB. I'm open to changing the naming though if you feel like this would be confusing. Can you provide some alternative names?

Reply:
I was thinking about something like `defaultNodes`, `defaultCurrencies` and `defaultPairs` but I am not opposed to the current naming scheme.

Reply:
Both look good from my perspective. @sangaman? @kilrau?

Reply:
Agree with @michael1011 `defaultNodes`, `defaultCurrencies` and `defaultPairs`